source/bsondump.txt

```diff
@@ -130,7 +129,7 @@ Options
 .. option:: --bsonFile
    
    Specifies a path to a :term:`BSON` file to dump to JSON. :option:`--bsonFile` is
-   an alternative to the positional :option:`<bsonFilename>` option.
+   an alternative to the positional ``<bsonFilename>`` option.
```

Comment:
• We still want to link to this, even though it's a positional argument (final param) as opposed to an explicit flag. The original is what we want, even though it looks strange:

```suggestion
   an alternative to the positional :option:`<bsonFilename>` option.
```